tests/models/test_huggingface_model.py

```diff
@@ -38,6 +39,50 @@ def saliency_gpt2_model():
     return inseq.load_model("gpt2", "saliency")
 
 
+def test_tokenizer_consistency(saliency_gpt2_model):
+    texts = EXAMPLES["texts"][2][0]
+    tok_model = inseq.load_model("gpt2", "saliency", tokenizer=AutoTokenizer.from_pretrained("gpt2", use_fast=False))
```

Comment:
Can we use `hf-internal-testing/tiny-random-gpt2` (if possible), or at least `distilgpt2` for this, since the test is not marked as slow?

Reply:
Sure, I will check their tokenizers to see if they have fast versions. But even now, the test is not really that slow since it only goes over three sentences for the attribution.

Reply:
I changed it to `distilpgt2`, the tokenizer of the tiny-model was too simple and was giving errors in the attribution (unrelated to this issue). But I am not sure if using the smaller model is actually faster since the `gpt2` model is downloaded as a fixture at the top of the test-file anyways and now the `distilpgt2` model has to be downloaded in addition just for this one test.

Reply:
The `gpt2` will be swapped out for `distilgpt2` in an upcoming PR, so let's stick to `distilgpt2`!

Reply:
ok, then everything should be ready once the checks have completed (no idea why they are so slow right now)